package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua

```diff
@@ -46,7 +46,10 @@ function M.is_outdoor_device()
 		M.get_model() == 'Ubiquiti UniFi-AC-MESH-PRO' then
 		return true
 
-	elseif M.match('ath79', 'generic', {'devolo,dvl1750x'}) then
+	elseif M.match('ath79', 'generic', {
+		'devolo,dvl1750x',
+		'nanostation-ac-wa',
```

Comment:
```suggestion
		'ubnt,nanostation-ac',
```

Reply:
Changed but can you tell me where I get this value from?

Reply:
That is the `board_name` used in `01_leds`, `02_network` et al.